hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/HazelcastSchemaUtils.java

```diff
@@ -68,11 +70,9 @@ public static HazelcastSchema createRootSchema(SqlCatalog catalog) {
             for (Map.Entry<String, Table> tableEntry : currentSchemaEntry.getValue().entrySet()) {
                 String tableName = tableEntry.getKey();
                 Table table = tableEntry.getValue();
-
-                HazelcastTable convertedTable = new HazelcastTable(
-                        table,
-                        createTableStatistic(table)
-                );
+                HazelcastTable convertedTable = table instanceof PartitionedMapTable
+                        ? new HazelcastTable(table, hz)
```

Comment:
for `PartitionedMapTable` we already have statistic in that object. Why do you started ignoring them?

Reply:
see `MapTableUtils.estimatePartitionedMapRowCount`

Reply:
>  we already have statistics in that object.

oh, see it. I compared them, and the existing version is less precise. Can rollback my solution.

Reply:
`MapTableUtils.estimatePartitionedMapRowCount` is less precise (especially for low-cardinality imaps) but does not require sending requests to other members each time we parse a query, so it is much cheaper

Reply:
https://github.com/hazelcast/hazelcast/pull/25249/commits/f65293f63779688a039ff62fe67a27ec787b7447